compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/test/BUILD

```diff
@@ -22,6 +22,7 @@ iree_lit_test_suite(
             "fill.mlir",
             "from_elements.mlir",
             "insert_slice.mlir",
+            "insert.mlir",
```

Comment:
Per the lint check: https://github.com/iree-org/iree/actions/runs/3695690035/jobs/6281665462, this change should also be made in the CMakeLists.txt file. You can run the Bazel -> CMake conversion script to do that for you: `./build_tools/bazel_to_cmake/bazel_to_cmake.py`